cli.js

```diff
@@ -4,8 +4,8 @@
 const argv = process.argv.slice(2)
 
 const opts = require('minimist')(argv, {
-  string: ['image', 'cwd'],
-  alias: { image: 'i' }
+  string: ['image', 'cwd', 'nodemodulespath'],
+  alias: { image: 'i'}
```

Comment:
Let's restore that whitespace (did you run `npm test` to lint?):

```suggestion
  alias: { image: 'i' }
```